packages/application-to-register-webapp/src/routes/developer/check-metric-file.js

```diff
@@ -0,0 +1,60 @@
+import constants from '../../utils/constants.js'
+import path from 'path'
+import { blobStorageConnector } from '@defra/bng-connectors-lib'
+
+const href = '#check-upload-correct-yes'
+const handlers = {
+  get: async (request, h) => {
+    const context = await getContext(request)
```

Comment:
no await needed, just reduce this down to `=> h.view(constants.views.DEVELOPER_CHECK_UPLOAD_METRIC, getContext(request)`